### DISCUSSION TITLE
Renove all changes in a single MR but commit per change

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [renovatebot/renovate#26725] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.135.0

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### What is your question?

Hello!

So, we used the `group:all` and it works great. I was wondering if there is a way to still group but every change of every package comes in a separate commit. As we have release automation, when it gets released the changes of more than 1 change per MR it just says the commit `chore(deps): update all dependencies` so it would be great to say something like `chore(deps): update dependency node to v20.11.0` but all grouped for all the changes within the same MR.

It does the `chore(deps): update dependency node to v20.11.0` when it is only one change which is great. Unfortunately,  we cannot do one MR per change as that is a lot of volume and we use self-hosted runners so not an ideal scenario.

Thank you!

P.S: As usual, Renovate rocks!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Not supported today. The FR is here: #7288
